src/app/components/table_details/table_details.js

```diff
@@ -76,13 +76,20 @@ angular
                 var is_ephemeral = !model.metadata;
                 var metadata = model.metadata || {};
 
+                var database;
+                if (model.database == null) { 
```

Comment:
This change will catch null, undefined, and empty-string database values
```suggestion
                if (!model.database) { 
```